### PR TITLE
Feature/#5 live preview

### PR DESCRIPTION
**Pull request by stefanmatar**
### Changed
- Added a preview button and a live updating preview screen
- You can now resize the preview panel
- Added buttons at the bottom of the preview panel that sets screen sizes automatically

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
               ]
             },
             "scripts": [
+              "node_modules/split.js/dist/split.min.js"
```
Comment (stefanmatar) [inline angular.json]: Split.js is used for the splitting of the editing and preview panel. It also has a drawer inbetween, so the user can decide on the size

File: projects/ixocreate/ngx-admin/src/lib/pipes/safe-url.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+
+@Pipe({
+  name: 'safeUrl'
+})
+export class SafeUrlPipe implements PipeTransform {
```
Comment (stefanmatar) [inline projects/ixocreate/ngx-admin/src/lib/pipes/safe-url.pipe.ts]: If you add a link in the src part of a html component, you need to sanitize it. This is a "| safeUrl" pipe to correct that automatically

Comment (luchaos) [reply projects/ixocreate/ngx-admin/src/lib/pipes/safe-url.pipe.ts]: Makes sense - sounds fine to me in this case. Let's make sure to use it with care :)

File: angular.json (hunk 1 above)
Comment (luchaos) [reply angular.json]: Awesome tool, i like how it works. Good choice!

**Pull request merged**